requrements/process.typ

```diff
@@ -0,0 +1,34 @@
+= Procesa apraksts
+
+Bakalaura darba IT atbalsta sistēma veido 3 procesus, saistībā ar bakalaura darbiem:
+- Bakalaura darba tēmas izvēle;
+- Bakalaura darba izstrāde;
+- Bakalaura darba iesniegšana un aizstāvēšana.
+
+@choose-theme atspoguļo bakalaura darba tēmas izvēles procesu. Vispirms, tiek nodefinēts darba vadītāju saraksts, kuri darba vadītāji grib palīdzēt ar bakalaura darbu izstrādi studentiem. IT sistēma šis process notiek automātiski ar mācībspēka paša piekrišanos ar dalību bakalaura darba izstrādē. Darba vadītājs var augšupielādēt sistēmā savus BD tēmu variantus, vai atstāt tukšu sarakstu, norādot, ka mācībspēks ir gatavs saņemt arī studentu tēmu piedāvājumus. Kad tēmu iesniegšanas termiņš ir beidzies, pēdējā kursa studenti saņem paziņojumu, ka var uzsākt ar darbu ar sistēmu. 
```

Comment:
.. tiek nodefinēts darba vadītāju saraksts, kuri darba vadītāji grib palīdzēt ar bakalaura darbu izstrādi studentiem. -> tiek nodefinēts saraksts ar darba vadītājiem, kuri grib palīdzēt studentiem ar bakalaura darba izstrādi.

IT sistēma šis process notiek automātiski ar mācībspēka paša piekrišanos ar dalību bakalaura darba izstrādē. -> IT sistēmā šis process notiek automātiski, kad mācībspēks piekrīt piedalīties bakalaura darba izstrādes procesā.

Darba vadītājs var augšupielādēt sistēmā .. -> Darba vadītājs var augšupielādēt vai pievienot sistēmā ..

.. var uzsākt ar darbu ar sistēmu. -> .. var uzsākt darbu ar sistēmu.